tools/mqtt-bench/templates/config.toml

```diff
@@ -1,26 +1,26 @@
 [mqtt]
+  timeout = 1000
```

Comment:
If this is `config.toml` that makes best performance, I suggest that we rename it to something like `reference.toml` or something, because I am afraid that someone will overwrite this config.

Also - I am not sure that in templates we really need a file called `config.toml` - this one must be produced manually by copying some of the templates for example in the root dir and then custommizing.